docs/customize_look&feel_guide.md

```diff
@@ -0,0 +1,39 @@
+# Customize the look & feel of Harbor
+
+The primary look & feel of Harbor supports to be customized with several simple steps. All the relevant customization configurations are saved in the `setting.json` file under `$HARBOR_DIR/src/ui_ng/src` folder with `json` format and will be loaded when the application is started to run. 
```

Comment:
All the relevant customization configurations are saved
==> All the relevant customization in configuration is saved

when the application is started to run.
==> when Harbor is launched.